danielforkner.Calculator/CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Newtonsoft.Json;
+using System.Linq;
+
+namespace CalculatorLibrary
```

Comment:
not mandatory but you can use the newest namespaces to better utilize space: 
![image](https://user-images.githubusercontent.com/108711476/219279649-cdefe019-fe98-49d2-adc3-f32fb4a74db7.png)

Reply:
Thank you, I will do that!